plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/Menu.js

```diff
@@ -110,9 +110,6 @@ Ext.define('NX.controller.Menu', {
         '#Permissions': {
           changed: me.refreshMenu
         },
-        '#State': {
-          changed: me.onStateChange
```

Comment:
@adreghiciu I removed a number of events which seemed to be redundant. For example, refreshing the app resulted in the 'featureselected' event being fired 13 times. By removing these events, I cut down on the redundancy, but since I don’t understand this code terribly well, it’s possible this has some negative effects. Let me know.